.github/workflows/webchat-CI.yml

```diff
@@ -22,7 +22,8 @@ jobs:
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Not specific to this PR, really, but we should move to `actions/checkout@v3` whenever we touch an action to avoid these warnings (and the eventual failure of our actions).
<img width="1647" alt="Screen Shot 2023-02-17 at 2 40 07 PM" src="https://user-images.githubusercontent.com/7569921/219773547-80805ef0-7d9d-4c8c-a597-38ca3dc9458d.png">

Reply:
Good to know. I can make that change in this PR